template/lambda-sqs-worker/src/services/aws.ts

```diff
@@ -3,7 +3,6 @@ import { Agent } from 'https';
 import { SNS } from 'aws-sdk';
 
 const agent = new Agent({
```

Comment:
Can we remove the agent altogether now?

Looks like `rejectUnauthorized` defaults to true anyway. https://nodejs.org/api/tls.html